packages/renderer/src/modules/rpc/index.ts

```diff
@@ -31,25 +40,25 @@ export function initRpc(iframe: HTMLIFrameElement, project: Project, cbs: Partia
   const params = { iframe, project, storage, camera };
 
   storage.handle('read_file', async ({ path }) => {
-    const file = await fs.readFile(await fs.resolve(project.path, path));
+    const file = await fs.readFile(await fs.resolve(await getBasePath(path, project), path));
```

Comment:
maybe we can also move the `await fs.resolve` inside `getBasePath`? so it's less verbose here